config/distribution/linux_aarch64_cpu.cmake

```diff
@@ -30,17 +30,13 @@ set(USE_F16C OFF CACHE BOOL "Build with x86 F16C instruction support")
 set(USE_DIST_KVSTORE OFF CACHE BOOL "Build with DIST_KVSTORE support")
 
 set(USE_MKLDNN ON CACHE BOOL "Build with MKL-DNN support")
-# Uncomment the following line to build MKLDNN with APL support
-# APL can be downloaded from https://developer.arm.com/tools-and-software/server-and-hpc/downloads/arm-performance-libraries
-# APL needs to be added to LD_LIBRARY_PATH
-## set(DNNL_BLAS_VENDOR “ARMPL” CACHE STRING “Build MKLDNN with Arm Performance Libraries as the BLAS library”)
-# Uncomment the following lines to build MKLDNN with ACL support
-# C++ 14 is requried to build with ACL and MKLDNN recommends building ACL from source rather than
-# using the pre-built binaries from https://github.com/ARM-software/ComputeLibrary/releases
-# If pre-built binaries are used anyways, make sure to copy and rename the appropriate binaries
-# folder from <acl_root>/lib/<binaries_folder> to <acl_root>/build
+# Pre-built binaries are available from  https://github.com/ARM-software/ComputeLibrary/releases
+# Make sure to copy and rename the appropriate binaries folder
+# from <acl_root>/lib/<binaries_folder_for_your_arch> to <acl_root>/build
```

Comment:
What happened to "MKLDNN recommends building ACL from source"?

Reply:
I removed it since this config file is used by us in ci/cd and we already know how to get pre-built binaries to work. Should I add this comment to `linux_arm.cmake`?

Reply:
I see. But if MKLDNN recommends building ACL from source, why do we use the prebuilt binaries? Is there any reason for their recommendation?

Reply:
"MKLDNN recommends building ACL from source" may not be 100% accurate sorry about this..
MKLDNN actually does not make this statement officially and I was originally referring to [this thread ](https://github.com/oneapi-src/oneDNN/issues/986#issuecomment-780845746). His original words are "Builds against Compute Library from OS package repos are not currently supported." MKLDNN's CMake files will look for a `build` folder in `ACL_ROOT_DIR`, and this makes me suspect that they have only tested with build-from-source ACL

Reply:
Cool, thanks for clarifying!